### DISCUSSION TITLE
Additional mods with server type of AUTO_CURSEFORGE

### DISCUSSION THREAD

**Discussion by DanAE111** [itzg/docker-minecraft-server#3075] [General]
I've got a server setup using AUTO_CURSEFORGE and the server spins up and downloads the mod pack no worries. 

However i want to download an additional resource from curseforge using the CF_FORCE_INCLUDE_MODS such as this one https://www.curseforge.com/minecraft/mc-mods/dcintegration the server seems to startup as normal but does not include the additional mod specified.

Is this intended behaviour or should this work?

**Comment by itzg** [comment]
Include additional mods from CurseForge by also using

https://docker-minecraft-server.readthedocs.io/en/latest/mods-and-plugins/curseforge-files/
